src/Apps/Order/Components/ShippingAddress.tsx

```diff
@@ -18,13 +20,20 @@ export const ShippingAddress = ({
   city,
   postalCode,
   region,
+  country,
 }: ShippingAddressProps) => {
-  const cityLine = city + (postalCode ? ` ${postalCode}` : "")
   return (
-    <Serif size="3t" style={{ whiteSpace: "pre-wrap" }}>
-      {[name, addressLine1, addressLine2, cityLine, region]
-        .filter(Boolean)
-        .join("\n")}
-    </Serif>
+    <>
+      <Serif size="3t">{name}</Serif>
+      <Serif size="3t">
+        {[addressLine1, (addressLine2 || "").trim()].filter(Boolean).join(", ")}
+      </Serif>
+      <Serif size="3t">
+        {city}, {region}, {postalCode}
+      </Serif>
+      <Serif size="3t">
+        {COUNTRY_CODE_TO_COUNTRY_NAME[country] || country}
```

Comment:
You could move this logic into a function `getCountryName(countryCode)` rather than using this big object directly everwhere?

Reply:
I think a hash lookup is simple enough for now.